apsuite/lattice_errors_new.py

```diff
@@ -0,0 +1,1539 @@
+        self._famdata = value
+
+    @property
+    def cutoff(self):
+        """Cut-off of the maximum values allowed for errors.
+
+        Returns:
+            float: in units of standard deviation.
+        """
+        return self._cutoff
+
+    @cutoff.setter
+    def cutoff(self, value):
+        self._cutoff = value
+
+    @property
+    def fam_errors(self):
+        """Dictionary containing the errors for all families.
+
+        Returns:
+            dictionary: The keys are: family_name, error_type and index.
+        """
+        return self._fam_errors
+
+    @fam_errors.setter
+    def fam_errors(self, value):
+        self._fam_errors = value
+
+    @property
+    def bba_idcs(self):
+        """Quadrupoles indexes where the bba will be performed.
+
+        Returns:
+            numpy array: array whose elements are the bba indexes.
+        """
+        return self._bba_idcs
+
+    @bba_idcs.setter
+    def bba_idcs(self, value):
+        self._bba_idcs = value
+
+    @property
+    def models(self):
+        """List if lattice models with errors.
+
+        Returns:
+            List of pymodels objects: list with nr_mach elements.
+        """
+        return self._models
+
+    @models.setter
+    def models(self, value):
+        self._models = value
+
+    @property
+    def nominal_model(self):
+        """Reference model to perform corrections.
+
+        Returns:
+            pymodels object: reference lattice.
+        """
+        return self._nominal_model
+
+    @nominal_model.setter
+    def nominal_model(self, value):
+        self._nominal_model = value
+
+    @property
+    def save_jacobians(self):
+        """Option to save the calculated jacobians.
+
+        Returns:
+            Boolean: If True the jacobians will be saved.
+        """
+        return self._save_jacobians
+
+    @save_jacobians.setter
+    def save_jacobians(self, value):
+        if type(value) != bool:
+            raise ValueError('Save jacobian must be bool type')
+        else:
+            self._save_jacobians = value
+
+    @property
+    def load_jacobians(self):
+        """Option to load jacobians.
+
+        Returns:
+            Boolean: If True the jacobians will be loaded.
+        """
+        return self._load_jacobians
+
+    @load_jacobians.setter
+    def load_jacobians(self, value):
+        if type(value) != bool:
+            raise ValueError('Load jacobian must be bool type')
+        else:
+            self._load_jacobians = value
+
+    @property
+    def orbcorr(self):
+        """Object of the class OrbitCorr.
+
+        Returns:
+            OrbitCorr object: Do orbit correction.
+        """
+        return self._orbcorr
+
+    @orbcorr.setter
+    def orbcorr(self, value):
+        self._orbcorr = value
+
+    @property
+    def orbcorr_params(self):
+        """Object of the class CorrParams.
+
+        Returns:
+            CorrParams object: Parameters used in orbit correction.
```

Comment:
in this case the path for the class should be specified in the doctring "apsuite.orbcorr.orbit_correction.CorrParams", otherwise the user would think this class is defined in this file